jablib/src/main/java/org/jabref/logic/bibtex/comparator/plausibility/PersonNamesPlausibilityComparator.java

```diff
@@ -0,0 +1,39 @@
+package org.jabref.logic.bibtex.comparator.plausibility;
+
+import org.jabref.logic.bibtex.comparator.ComparisonResult;
+import org.jabref.model.entry.AuthorList;
+
+public class PersonNamesPlausibilityComparator implements FieldValuePlausibilityComparator {
+    @Override
+    public ComparisonResult compare(String left, String right) {
+
+        if (left == null || right == null) {
+            return ComparisonResult.UNDETERMINED;
+        }
```

Comment:
Sure that one of it can be `null`?

I bet that both are non-null - and you can annotate the whole class as `@Nullmarked` (because nothing is null)